component/tasks.js

```diff
@@ -31,6 +31,13 @@ class Tasks {
     model.setProperty(propertyName, property);
     cb(null, propertyDef);
   }
+  addModelRelation(relationName, fromModelId, toModelId, data, cb) {
+    const workspace = this;
+    const model = workspace.getModel(fromModelId);
+    const relation = model.addRelation(relationName, toModelId, data);
+    model.setRelation(relationName, relation);
+    writeOperations.writeModel(model, cb);
```

Comment:
I think we should make your helper util lib name more obvious -- like `fsWriter.writeModel(model, cb)` to make it obvious you're writing to the filesystem.